frontend/src/components/viewers/Tensorboard.test.tsx

```diff
@@ -102,9 +103,14 @@ describe('Tensorboard', () => {
     const config = { type: PlotType.TENSORBOARD, url: 'http://test/url' };
     const getAppMock = () => Promise.resolve({ podAddress: 'test/address', tfVersion: '1.14.0' });
     jest.spyOn(Apis, 'getTensorboardApp').mockImplementation(getAppMock);
+    jest.spyOn(Apis, 'isTensorboardPodReady').mockImplementation(() => Promise.resolve(true));
     tree = shallow(<TensorboardViewer configs={[config]} />);
 
     await TestUtils.flushPromises();
+    jest.runOnlyPendingTimers();
```

Comment:
Curious. What does this do?

Reply:
Just want to trigger one run of the task I set up in SetInterval, in order to match the snapshot (the last line in this test) because the notice message is in the snapshot and the notice message varies depending on the result of the run (result is true, notice message is empty; result is false, notice message is blahblah).